vm/src/dictdatatype.rs

```diff
@@ -25,10 +34,10 @@ impl<T> Default for Dict<T> {
 }
 
 #[derive(Clone)]
-struct DictEntry<T> {
-    hash: usize,
-    key: PyObjectRef,
-    value: T,
+pub struct DictEntry<T> {
```

Comment:
I don't think we should be exposing the internals of the dictionary implementation here, this should be private. I think that we just need ```delete_or_insert``` and maybe ```delete_if_exists``` instead of  ```unchecked_*```

Reply:
Fixed